src/main/java/com/rocketden/main/service/GameManagementService.java

```diff
@@ -38,21 +46,54 @@ protected GameManagementService(RoomRepository repository, SocketService socketS
         currentGameMap = new HashMap<>();
     }
 
-    public Game createAddGameFromRoom(Room room) {
-        // TODO: Create the game from the room (or roomId).
-        Game game = new Game();
-        currentGameMap.put(room.getRoomId(), game);
-        return null;
-    }
-
     protected Game getGameFromRoomId(String roomId) {
-        return currentGameMap.get(roomId);
+        Game game = currentGameMap.get(roomId);
+        if (game == null) {
+            throw new ApiException(GameError.NOT_FOUND);
+        }
+
+        return game;
     }
 
     protected void removeGame(String roomId) {
         currentGameMap.remove(roomId);
     }
 
+    public GameDto getGameDto(String roomId) {
```

Comment:
nit: Maybe change to `getGameDtoFromRoomId`, or change the above analogous method to just `getGame`, to keep some uniformity across method names?